sdcm/cluster.py

```diff
@@ -931,15 +979,43 @@ def start_alert_manager_thread(self):
     def silence_alert(self, alert_name, duration=None, start=None, end=None):
         return AlertSilencer(self._alert_manager, alert_name, duration, start, end)
 
+    def _dc_info_str(self):
+        dc_info = []
+
+        # Example: `ManagerPodCluser` - the `params` is not needed and may be not passed there. Manager always created in the first DC
+        if not hasattr(self.parent_cluster, "params"):
+            TestFrameworkEvent(source=self.__class__.__name__,
+                               message="The parent cluster has not 'params' attribute",
+                               trace=sys._getframe().f_back,  # pylint: disable=protected-access
+                               severity=Severity.ERROR).publish()
```

Comment:
Why do we raise "Error" SCT event? ManagerPodCluster doesn't use params itself.
So, I disagree that it should raise error in this case

Reply:
We want to find all placed that have no appropriate "params" attribute. If warning will be raised, it won't be found, nobody check warnings.
About ManagerPodCluster - I added there "params"

Reply:
All cluster objects should have `params`, since base is using it (or at least now it's using it)

we have more occurences that assumes it, but it was like in this case when every single one needs to have it